client/.babelrc

```diff
@@ -4,7 +4,7 @@
       "@babel/preset-env",
       {
         "exclude": ["@babel/plugin-transform-async-to-generator", "@babel/plugin-transform-arrow-functions"],
-        "corejs": "2",
+        "corejs": "3",
```

Comment:
I don't understand any of the JS dependency related changes; @eradman would you mind taking a look? Are we able to change these deps to enable arm64 images?